### DISCUSSION TITLE
ldm.models

### DISCUSSION THREAD

**Discussion by nolbertocastro** [TheLastBen/fast-stable-diffusion#1670] [Q&A]
Hi, I'm having this problem, when I run the last step in the notebook

'ModuleNotFoundError: No module named 'ldm.models.diffusion.sampling_util'

**Comment by TheLastBen** [comment]
Try running the colab without the first cell, if it works, the use a clean SD folder

**Reply by nolbertocastro** [reply]
hi, the first cell would be the google drive connection, you mean the install/update cell?

**Reply by nolbertocastro** [reply]
RuntimeError: PytorchStreamReader failed reading zip archive: failed finding central directory


Stable diffusion model failed to load, exiting

I uploaded my custom model manually, but now I have this problem

**Reply by TheLastBen** [reply]
the model is corrupt

**Reply by nolbertocastro** [reply]
I'm trying to use the Automatic1111 colab notebook, is it a general error?

**Reply by TheLastBen** [reply]
rename/remove your sd folder and use the latest notebook, you shouldn't get those errors

**Reply by nolbertocastro** [reply]
thanks, is it better to upload the custom model manually, or I could connect the notebook to google drive?

**Reply by TheLastBen** [reply]
connect the notebook to your gdrive there is no problem loading models from gdrive

**Reply by nolbertocastro** [reply]
okey, thank you so much
